### DISCUSSION TITLE
Is there a VSCode plugin that hides long classes?

### DISCUSSION THREAD

**Discussion by martinszeltins** [tailwindlabs/tailwindcss#5516] [Help]
Is there a plugin that can hide my classes and instead show something like this? And when you click on the 3 ... it will expand and show all the classes. This would make my html look much cleaner :)

````
<div class="...">
  <div class="...">content</div>
</div>
````

instead of this

````
<div class="fixed inset-0 flex items-end justify-center px-4 py-6 pointer-events-none sm:p-6 sm:items-start sm:justify-end">
  <div class="max-w-sm w-full bg-white shadow-lg rounded-lg pointer-events-auto">content</div>
</div>
````

**Comment by gchartier** [comment]
I don't think there is but I just had this idea while hearing someone complain that tailwind clutters the markup. It would be nice to have my tooling fix that.

**Comment by burrijw** [comment]
For anyone else who ends up here, there is an extension that accomplishes this: https://marketplace.visualstudio.com/items?itemName=moalamri.inline-fold

**Reply by martinszeltins** [reply]
> For anyone else who ends up here, there is an extension that accomplishes this: [marketplace.visualstudio.com/items?itemName=moalamri.inline-fold](https://marketplace.visualstudio.com/items?itemName=moalamri.inline-fold)

This is perfect, exactly what I was looking for. Looks like it was recently created.

**Reply by burrijw** [reply]
Someone highlighted it on TikTok (which I forgot to save). I've been looking for it ever since and finally found it. 😆

**Reply by jnthn223** [reply]
> Someone highlighted it on TikTok (which I forgot to save). I've been looking for it ever since and finally found it. 😆

I found it on youtube shorts and forgot the name of it. Thanks :)
